packages/applications-service-api/__tests__/unit/controllers/representations.test.js

```diff
@@ -70,42 +78,37 @@ const returnData = {
   itemsPerPage: 3,
   totalPages: 1,
   currentPage: 1,
+  filters: { typeFilters: [] },
 };
 
-jest.mock('../../../src/models', () => {
-  // eslint-disable-next-line global-require
-  const SequelizeMock = require('sequelize-mock');
-  const dbMock = new SequelizeMock();
-  const Representation = dbMock.define('Representation');
-
-  Representation.$queryInterface.$useHandler((query, queryOptions) => {
-    if (queryOptions[0].where.CaseReference === 'EN010009') {
-      return mockData;
-    }
-    if (queryOptions[0].where.CaseReference === 'EN000000') {
-      return { rows: [], count: 0 };
-    }
-    const symbolKey = Reflect.ownKeys(queryOptions[0].where).find(
-      (key) => key.toString() === 'Symbol(and)'
-    );
-    if (queryOptions[0].where[symbolKey][0].CaseReference === 'EN010009') {
-      return mockData;
-    }
-    return null;
-  });
-  const db = {
-    Representation,
-  };
-  return db;
-});
-
 jest.mock('../../../src/lib/config.js', () => ({
   logger: {
     level: process.env.LOGGER_LEVEL || 'info',
   },
   itemsPerPage: 3,
 }));
 
+jest.mock('../../../src/services/representation.service');
```

Comment:
I appreciate it is much easier to mock the servicer due to the complexity of mocking Sequelize.  However, we are now missing a test coverage for the representation business service.  Happy for that to be a separate unit test file, but I think we shouldn't avoid it due to the difficulty.  Let's just find a solution together to manage the complexity.

Reply:
What do you think @lvnhmd ?

Reply:
@cpcundill - I am working on it. Opted for a separate unit test file.